Sources/ComposableArchitecture/Store.swift

```diff
@@ -1,5 +1,6 @@
 import Combine
 import Foundation
+import SwiftUI
```

Comment:
Having to import all of SwiftUI in Store.swift is unfortunate. I'm not quite sure what the house style is so please do let me know if you'd prefer me to import just Animation and Transaction, or move these functions to an extension in a separate file instead